src/scss/_components.table-of-contents.scss

```diff
@@ -0,0 +1,77 @@
+.cmp-table-of-contents {
+  padding: 15rem 1rem;
+
+  &__items {
+    max-width: 71rem;
+    margin: 0 auto;
+    padding-top: 6rem;
+  }
+
+  &__list {
+    @supports (display: grid) {
+      @media (min-width: $bp-md) {
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+        grid-gap: 7rem;
+        justify-content: center;
+      }
+
+      @media (min-width: $bp-xl) {
+        grid-template-columns: 27rem 25rem;
+        grid-gap: 10rem;
+      }
+    }
+
+    counter-reset: count;
+    list-style: none;
+  }
+
+  &__grid-child {
+    @supports (display: grid) {
+      @media (min-width: $bp-md) {
+        display: grid;
+        grid-gap: 4.5rem;
+        grid-column: span 1;
+        align-items: center;
+      }
+    }
+  }
+
+  &__item {
+    counter-increment: count;
+    position: relative;
+    padding-left: 2rem;
+    max-width: 25rem;
+    margin: 0 auto 3rem;
+
+    @supports (display: grid) {
+      @media (min-width: $bp-md) {
+        margin: 0;
+      }
+    }
+
+    &::before {
+      font-family: $font-script;
+      text-align: center;
+      font-size: 6.25rem;
+      color: $blue;
+      transform: rotate(-10deg);
+      text-shadow:
+        0 0 0.075em rgba($blue, 0.6),
+        0 0.02em 0 darken($blue, 32%),
+        -0.0075em 0.005em 0 $white,
+        0.01em 0.005em 0 $white,
+        0 0 0.01em $black
+        ;
+      content: counter(count);
+      display: inline-block;
+      position: absolute;
```

Comment:
Gonna throw a wrench in this one. 🙁 This will need to be done without `position: absolute`. The text of the TOC item should be vertically centered, so the `&__item` will need to be `display: flex` with `align-items: center` to vertically align the section number with the title.

Reply:
Ah, yes, that's a much better way to vertically align everything 👍 How do you recommend we take care of the space between the number and the item? We were accomplishing this with absolute positioning and `right`, which still works with the flex properties added. Should we keep the positioning to accomplish this?

Reply:
`margin-right` on the number pseudo element should take care of the positioning.